Description: libxl: Fix test data for emulator and loader
 For the loader part this is a bit of a brute force hack. The libxl
 driver is build with LIBXL_FIRMWARE_DIR correctly set (which includes
 a version part). But the test data seems static.
 The emulator (at least for us) is always /usr/bin/qemu-system-i386.
Forwarded: no
Author: Stefan Bader <stefan.bader@canonical.com>

Index: libvirt-1.3.1/tests/xlconfigdata/test-fullvirt-direct-kernel-boot.cfg
===================================================================
--- libvirt-1.3.1.orig/tests/xlconfigdata/test-fullvirt-direct-kernel-boot.cfg	2016-01-10 02:57:37.000000000 +0100
+++ libvirt-1.3.1/tests/xlconfigdata/test-fullvirt-direct-kernel-boot.cfg	2016-01-29 17:41:06.228815963 +0100
@@ -13,7 +13,7 @@ localtime = 0
 on_poweroff = "destroy"
 on_reboot = "restart"
 on_crash = "restart"
-device_model = "/usr/lib/xen/bin/qemu-system-i386"
+device_model = "/usr/bin/qemu-system-i386"
 sdl = 0
 vnc = 1
 vncunused = 1
Index: libvirt-1.3.1/tests/xlconfigdata/test-fullvirt-direct-kernel-boot.xml
===================================================================
--- libvirt-1.3.1.orig/tests/xlconfigdata/test-fullvirt-direct-kernel-boot.xml	2016-01-10 02:57:37.000000000 +0100
+++ libvirt-1.3.1/tests/xlconfigdata/test-fullvirt-direct-kernel-boot.xml	2016-01-29 17:37:50.028818988 +0100
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='x86_64' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <kernel>/tmp/vmlinuz</kernel>
     <initrd>/tmp/initrd</initrd>
     <cmdline>ignore_loglvl</cmdline>
@@ -22,7 +22,7 @@
   <on_reboot>restart</on_reboot>
   <on_crash>restart</on_crash>
   <devices>
-    <emulator>/usr/lib/xen/bin/qemu-system-i386</emulator>
+    <emulator>/usr/bin/qemu-system-i386</emulator>
     <disk type='block' device='disk'>
       <driver name='phy' type='raw'/>
       <source dev='/dev/HostVG/XenGuest2'/>
Index: libvirt-1.3.1/tests/xlconfigdata/test-fullvirt-multiusb.cfg
===================================================================
--- libvirt-1.3.1.orig/tests/xlconfigdata/test-fullvirt-multiusb.cfg	2016-01-10 02:57:37.000000000 +0100
+++ libvirt-1.3.1/tests/xlconfigdata/test-fullvirt-multiusb.cfg	2016-01-29 17:41:23.180815701 +0100
@@ -13,7 +13,7 @@ localtime = 0
 on_poweroff = "destroy"
 on_reboot = "restart"
 on_crash = "restart"
-device_model = "/usr/lib/xen/bin/qemu-dm"
+device_model = "/usr/bin/qemu-system-i386"
 sdl = 0
 vnc = 1
 vncunused = 1
Index: libvirt-1.3.1/tests/xlconfigdata/test-fullvirt-multiusb.xml
===================================================================
--- libvirt-1.3.1.orig/tests/xlconfigdata/test-fullvirt-multiusb.xml	2016-01-10 02:57:37.000000000 +0100
+++ libvirt-1.3.1/tests/xlconfigdata/test-fullvirt-multiusb.xml	2016-01-29 17:38:33.588818316 +0100
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='x86_64' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
@@ -19,7 +19,7 @@
   <on_reboot>restart</on_reboot>
   <on_crash>restart</on_crash>
   <devices>
-    <emulator>/usr/lib/xen/bin/qemu-dm</emulator>
+    <emulator>/usr/bin/qemu-system-i386</emulator>
     <disk type='block' device='disk'>
       <driver name='phy' type='raw'/>
       <source dev='/dev/HostVG/XenGuest2'/>
Index: libvirt-1.3.1/tests/xlconfigdata/test-new-disk.cfg
===================================================================
--- libvirt-1.3.1.orig/tests/xlconfigdata/test-new-disk.cfg	2016-01-10 02:57:37.000000000 +0100
+++ libvirt-1.3.1/tests/xlconfigdata/test-new-disk.cfg	2016-01-29 17:41:47.112815332 +0100
@@ -13,7 +13,7 @@ localtime = 0
 on_poweroff = "destroy"
 on_reboot = "restart"
 on_crash = "restart"
-device_model = "/usr/lib/xen/bin/qemu-dm"
+device_model = "/usr/bin/qemu-system-i386"
 sdl = 0
 vnc = 1
 vncunused = 1
Index: libvirt-1.3.1/tests/xlconfigdata/test-new-disk.xml
===================================================================
--- libvirt-1.3.1.orig/tests/xlconfigdata/test-new-disk.xml	2016-01-10 02:57:37.000000000 +0100
+++ libvirt-1.3.1/tests/xlconfigdata/test-new-disk.xml	2016-01-29 17:39:02.844817865 +0100
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='x86_64' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
@@ -19,7 +19,7 @@
   <on_reboot>restart</on_reboot>
   <on_crash>restart</on_crash>
   <devices>
-    <emulator>/usr/lib/xen/bin/qemu-dm</emulator>
+    <emulator>/usr/bin/qemu-system-i386</emulator>
     <disk type='block' device='disk'>
       <driver name='phy' type='raw'/>
       <source dev='/dev/HostVG/XenGuest2'/>
Index: libvirt-1.3.1/tests/xlconfigdata/test-spice-features.cfg
===================================================================
--- libvirt-1.3.1.orig/tests/xlconfigdata/test-spice-features.cfg	2016-01-10 02:57:37.000000000 +0100
+++ libvirt-1.3.1/tests/xlconfigdata/test-spice-features.cfg	2016-01-29 17:42:24.340814759 +0100
@@ -13,7 +13,7 @@ localtime = 0
 on_poweroff = "destroy"
 on_reboot = "restart"
 on_crash = "restart"
-device_model = "/usr/lib/xen/bin/qemu-dm"
+device_model = "/usr/bin/qemu-system-i386"
 vif = [ "mac=00:16:3e:66:92:9c,bridge=xenbr1,script=vif-bridge,model=e1000" ]
 parallel = "none"
 serial = "none"
Index: libvirt-1.3.1/tests/xlconfigdata/test-spice-features.xml
===================================================================
--- libvirt-1.3.1.orig/tests/xlconfigdata/test-spice-features.xml	2016-01-10 02:57:37.000000000 +0100
+++ libvirt-1.3.1/tests/xlconfigdata/test-spice-features.xml	2016-01-29 17:39:43.812817234 +0100
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='x86_64' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
@@ -19,7 +19,7 @@
   <on_reboot>restart</on_reboot>
   <on_crash>restart</on_crash>
   <devices>
-    <emulator>/usr/lib/xen/bin/qemu-dm</emulator>
+    <emulator>/usr/bin/qemu-system-i386</emulator>
     <disk type='block' device='disk'>
       <driver name='phy' type='raw'/>
       <source dev='/dev/HostVG/XenGuest2'/>
Index: libvirt-1.3.1/tests/xlconfigdata/test-spice.cfg
===================================================================
--- libvirt-1.3.1.orig/tests/xlconfigdata/test-spice.cfg	2016-01-10 02:57:37.000000000 +0100
+++ libvirt-1.3.1/tests/xlconfigdata/test-spice.cfg	2016-01-29 17:42:07.696815015 +0100
@@ -13,7 +13,7 @@ localtime = 0
 on_poweroff = "destroy"
 on_reboot = "restart"
 on_crash = "restart"
-device_model = "/usr/lib/xen/bin/qemu-dm"
+device_model = "/usr/bin/qemu-system-i386"
 vif = [ "mac=00:16:3e:66:92:9c,bridge=xenbr1,script=vif-bridge,model=e1000" ]
 parallel = "none"
 serial = "none"
Index: libvirt-1.3.1/tests/xlconfigdata/test-spice.xml
===================================================================
--- libvirt-1.3.1.orig/tests/xlconfigdata/test-spice.xml	2016-01-10 02:57:37.000000000 +0100
+++ libvirt-1.3.1/tests/xlconfigdata/test-spice.xml	2016-01-29 17:40:11.108816813 +0100
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='x86_64' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
@@ -19,7 +19,7 @@
   <on_reboot>restart</on_reboot>
   <on_crash>restart</on_crash>
   <devices>
-    <emulator>/usr/lib/xen/bin/qemu-dm</emulator>
+    <emulator>/usr/bin/qemu-system-i386</emulator>
     <disk type='block' device='disk'>
       <driver name='phy' type='raw'/>
       <source dev='/dev/HostVG/XenGuest2'/>
Index: libvirt-1.3.1/tests/xlconfigdata/test-vif-rate.cfg
===================================================================
--- libvirt-1.3.1.orig/tests/xlconfigdata/test-vif-rate.cfg	2016-01-10 02:57:37.000000000 +0100
+++ libvirt-1.3.1/tests/xlconfigdata/test-vif-rate.cfg	2016-01-29 17:42:59.828814211 +0100
@@ -13,7 +13,7 @@ localtime = 0
 on_poweroff = "destroy"
 on_reboot = "restart"
 on_crash = "restart"
-device_model = "/usr/lib/xen/bin/qemu-dm"
+device_model = "/usr/bin/qemu-system-i386"
 sdl = 0
 vnc = 1
 vncunused = 1
Index: libvirt-1.3.1/tests/xlconfigdata/test-vif-rate.xml
===================================================================
--- libvirt-1.3.1.orig/tests/xlconfigdata/test-vif-rate.xml	2016-01-10 02:57:37.000000000 +0100
+++ libvirt-1.3.1/tests/xlconfigdata/test-vif-rate.xml	2016-01-29 17:40:42.252816332 +0100
@@ -6,7 +6,7 @@
   <vcpu placement='static'>1</vcpu>
   <os>
     <type arch='x86_64' machine='xenfv'>hvm</type>
-    <loader type='rom'>/usr/lib/xen/boot/hvmloader</loader>
+    <loader type='rom'>/usr/lib/xen-4.6/boot/hvmloader</loader>
     <boot dev='cdrom'/>
   </os>
   <features>
@@ -19,7 +19,7 @@
   <on_reboot>restart</on_reboot>
   <on_crash>restart</on_crash>
   <devices>
-    <emulator>/usr/lib/xen/bin/qemu-dm</emulator>
+    <emulator>/usr/bin/qemu-system-i386</emulator>
     <disk type='block' device='disk'>
       <driver name='phy' type='raw'/>
       <source dev='/dev/HostVG/XenGuest2'/>
